### PR TITLE
I started by adding some requested items to loadouts and it got out of hand (cleans up BO/QM/Tox/Xeno jobs + sprites a lot)

### PR DESCRIPTION
**Pull request by MrMelbert**
Atomization be damned, we live in a society.

#### - Cleans up the Bridge Officer's outfit and job datum (again), gives them a new PDA sprite + hat sprite
image
- BO now has their own PDA + sprite - required a few non-modular changes but they're mostly all clean.
- BO now has a non-bowman version of their headset properly defined (not used in-game)
- BO's beret is now armored and unique to them (with a unique sprite)
- BO's armor vest has changed sprite, again, and is now slightly more armored
- BO's suit can no longer be adjusted, as the sprite has no adjusted sprite.
- Changed BO typepath to Bridge_officer for clean code

#### - Cleans up the Quartermaster to head of staff transition
- HoP no longer has supply radio on their headset
- QM's headset has supply radio properly now (was bugged)
- QM's headset uses a command headset sprite and has a unique name
- QM's encryption key uses the unused QM key sprite now
- A bunch of stuff in the QM job datum now properly denotes them as a head of staff (all non-module changes but the file was already touched so it doesn't matter much)
- QM starts with an advanced tablet preset instead of a cargo one

#### - Fixes some missing sprites for toxicologist and xenobiologist labcoats
- Tox labcoat and xenobio labcoat used their on-mob sprites for their obj icons, this has been rectified and they have icons for their obj states now.

#### - Loadout changes, since that's what it all started out as
- Assistant formal moved to formal suits
- Skateboard added to inhand items
- Card binders, Kotahi decks, Dice bags, Wizoff decks, and Playing cards added to misc items

### CODE REVIEW

File: jollystation_modules/code/game/objects/items/devices/radio/encryptionkey.dm [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 /obj/item/encryptionkey/heads/qm
 	name = "\proper the quartermaster's encryption key"
-	icon_state = "cargo_cypherkey"
-	channels = list(RADIO_CHANNEL_CARGO = 1, RADIO_CHANNEL_COMMAND = 1)
-	
+	icon_state = "qm_cypherkey"
+	channels = list(RADIO_CHANNEL_SUPPLY = 1, RADIO_CHANNEL_COMMAND = 1)
+
+/obj/item/encryptionkey/heads/hop
+	channels = list(RADIO_CHANNEL_SERVICE = 1, RADIO_CHANNEL_COMMAND = 1)
```
Comment (MrMelbert) [inline jollystation_modules/code/game/objects/items/devices/radio/encryptionkey.dm:12]: Do you hear the people sing? Singing the song of angry men. It is the music of the people who will not be slaves again...

**Comment by MrMelbert** [comment]
Don’t merge this yet I was just about to push a commit and I had an internet outage

**Review by Jolly-66 — CHANGES_REQUESTED** [review]
i'm not really vibing with the beret, 
i think the general color scheme we we're going to aim for was like this grayish-black/navy  blue/gold/red trim kinda thing, i'd reccomend changing the beret to be like hos/warden, blue with gold (i forget whos it is but know its between one of the two)
if you're gonna use blue for bo try to not make it cyan/bright blue

**Comment by Jolly-66** [comment]
i should mention everything else is fine

**Comment by MrMelbert** [comment]
the beret was just copy+pasted HoS beret with the insignia changed from gold to blue, i'm not an artist

**Comment by Jolly-66** [comment]
ok give me the dmi i'll do it myself

insert thanos gif here

**Comment by MrMelbert** [comment]
You can download the DMI directly from the PR
image
image

**Review by Jolly-66 — APPROVED** [review]

**Pull request merged**